app/views/api/umgap/casestudy_preconfigured.html.erb

```diff
@@ -0,0 +1,137 @@
+      <div class="card-supporting-text">
+        <p>
+          The preconfigured pipelines require some databases and external tools.
+          The setup script will help you to get these in the right place.
+          If you're planning to use the tryptic precision, tryptic sensitivity, high precision or max precision pipelines,
+           you'll need to <a href="https://github.com/unipept/FragGeneScanPlusPlus/">install FragGeneScan++</a>.
+          In this casestudy, we will use the two tryptic pipelines, so we don't have to download the (large) 9-mer index.
+        </p>
+        <pre style="max-height: initial;">
+<!--   --><b>$</b> umgap-setup.sh -f /opt/FragGeneScanPlusPlus
+<!--   -->Use '/home/ninewise/.config/unipept' as configuration directory? [y]/n y
+<!--   -->Created directory /home/ninewise/.config/unipept.
+<!--   -->Found, tested and remembered the FragGeneScan++ location.
+<!--   -->Use '/home/ninewise/.local/share/unipept' as data directory? [y]/n y
+<!--   -->Created directory /home/ninewise/.local/share/unipept.
+<!--   -->Checking the latest version on the server.
+<!--   -->Latest version is 2020-12-02.
+<!--   -->
+<!--   -->For any type of analysis, you need a taxonomony. For mapping tryptic or
+<!--   -->9-mer peptides to it, you need the respective index file of the same
+<!--   -->version.
+<!--   -->
+<!--   -->Would you like to download the taxonomy from 2020-12-02 (115MiB)? [y]/n y
+<!--   -->Would you like to download the tryptic index from 2020-12-02 (12GiB)? [y]/n y
+<!--   -->Would you like to download the 9-mer index from 2020-12-02 (152GiB)? [y]/n n
+<!--   -->  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+<!--   -->                                 Dload  Upload   Total   Spent    Left  Speed
+<!--   -->100  114M  100  114M    0     0  6477k      0  0:00:18  0:00:18 --:--:-- 6332k
+<!--   -->  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+<!--   -->                                 Dload  Upload   Total   Spent    Left  Speed
+<!--   -->100   12G  100   12G    0     0  5076k      0  0:34:12  0:34:12 --:--:-- 5790k</pre>
+      </div>
+      <dl class='dl-horizontal'>
+        <dt><samp>-c dir</samp></dt><dd>Set a different configuration directory.</dd>
+        <dt><samp>-f dir</samp></dt><dd>Link the location of your FragGeneScan installation.</dd>
+        <dt><samp>-d dir</samp></dt><dd>Set a different location for downloading the database files. This can be on a separate disk.</dd>
+        <dt><samp>-y</samp></dt><dd>Download all files of the latest version without asking for confirmation.</dd>
+      </dl>
+    </div>
+
+    <div class="card">
+      <div class="card-title card-title-colored"><h2 class="card-title-text">Analyse</h2></div>
+      <div class="card-supporting-text">
+        <p>
+          When setup is complete, the analyse script will automatically use the downloaded databases and linked tools.
```

Comment:
The 2 analyses are run with a single command, but reality this will probably never happen. I would only do a single analysis here to prevent causing confusion (this will also reduce the number of args and make it easier to understand). 

I would also show a bit of the input file. This really helps to clear up any potential confusion of the format of the input file. The same for the output file.